tests/src/test/scala/ch/renku/acceptancetests/tooling/ScreenCapturing.scala

```diff
@@ -0,0 +1,24 @@
+package ch.renku.acceptancetests.tooling
+
+import java.io.File
+import java.nio.file.Paths
+import java.time.LocalDateTime.now
+import java.time.format.DateTimeFormatter.ofPattern
+
+import org.scalatestplus.selenium.{Driver, WebBrowser}
+import org.openqa.selenium.WebDriver
+
+trait ScreenCapturing {
+
+  /**
+    * Public helper function to write a screenshot of the current browser state.
+    */
+  def writeScreenshot(implicit browser: WebBrowser with Driver, webDriver: WebDriver) = {
```

Comment:
Wouldn't `takeScreenshot` read better here?

Reply:
Good suggestion. Made this change in #732